src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -18,10 +18,12 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetStandard)'">
     <DefineConstants>$(DefineConstants);NETSTANDARD_RUNTIME</DefineConstants>
   </PropertyGroup>
+  <ItemGroup>

```

Comment:
Is this needed ? I don't see any resources in the Exceptions folder.

Reply:
no, this is VS being "helpful" again.  fixed